File: BE/src/main/java/com/projekt/klinikaStudyBase/dto/VizsgalatiModDetailDto.java
package com.projekt.klinikaStudyBase.dto;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@Builder
@AllArgsConstructor
@NoArgsConstructor
public class VizsgalatiModDetailDto {
    private Long id;
    private String hivatkozasiNev;
    private String megnevezes;
    private Double eur;
    private Double huf;
    //private String megjegyzes; TODO
}

File: FE/klinikaStudyBase/src/app/dto/vizsgalatiModDetailDto.ts
export class VizsgalatiModDetailDto {

  id: number;
  hivatkozasiNev: string;
  megnevezes: string;
  eur: string;
  huf: string;

  constructor(id: number, hivatkozasiNev: string, megnevezes: string, eur: string, huf: string) {
    this.id = id;
    this.hivatkozasiNev = hivatkozasiNev;
    this.megnevezes = megnevezes;
    this.eur = eur;
    this.huf = huf;
  }

}
